doc/examples/ikfast/ikfast_tutorial.rst

```diff
@@ -34,32 +29,7 @@ You should have a MoveIt configuration package for your robot that was created b
 OpenRAVE is a planning framework as complex as MoveIt itself and installing it is tricky -- particularly because its public documentation is not maintained anymore.
 Fortunately, personalrobotics provide a `docker image <https://hub.docker.com/r/personalrobotics/ros-openrave>`_ based on Ubuntu 14.04 with OpenRAVE 0.9.0 and ROS Indigo installed, which can be used to generate the solver code once.
 
-So the easiest way to run the IKFast code generator is through this docker image.
-For manual building instructions (tailored towards Ubuntu 16.04), please see the `Kinetic version of this tutorial <http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/ikfast/ikfast_tutorial.html>`_.
-To follow the recommended, docker-based approach, ensure you have docker installed and started: ::
-
- sudo apt-get install docker.io
- sudo service docker start
-
-The following command will ensure that you can run docker with your user account (adding $USER to the docker group): ::
-
- sudo usermod -a -G docker $USER
-
-You need to log off/log on in order to actually activate this permission change.
-
-Install the MoveIt IKFast package either from Debian packages or from source.
-
-**Binary Install**: ::
-
- sudo apt-get install ros-${ROS_DISTRO}-moveit-kinematics
-
-**Source**
-
-Inside your catkin workspace's ``./src`` directory: ::
-
- git clone https://github.com/moveit/moveit.git
- rosdep install -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO}
- catkin build
+The script below downloads this docker image and starts it automatically.
```

Comment:
Which script below? Is it the `auto_create_ikfast_moveit_plugin.sh`?

If so, I might rephrase to something like "The commands provided in this tutorial automatically download and start this Docker image, so no extra steps are needed for now."